core/src/main/java/org/everit/json/schema/Schema.java

```diff
@@ -221,4 +238,12 @@ protected ValidationException failure(Class<?> expectedType, Object actualValue)
     protected boolean canEqual(final Object other) {
         return (other instanceof Schema);
     }
+
+    protected boolean checkNullity(final Object subject) {
```

Comment:
This implementation looks odd to me. This should be a void method and it should do something like `throw failure("value is null", "nullable");` instead of returning false.

I played around with it locally a bit, and it seems the `SchemaLoaderTest#nullableTestError()` passes only by accident (the exception message is "expected type: String, found: Null"), and wouldn't work for schemas which don't declare their `"type"` explicitly (which is valid in json schema).

Please change this method to throw an exception instead of returning `false`, and also write some unittests for each Schema subclasses using it (as mentioned below).

Reply:
There's no other way to do that. I've tried to do as you wish, but in fact I need a way to "skip" the remaining validation process after I've found that something is nullable and is null. If the validation goes through, It gives an error when it checks the type (it founds type NULL, expecting for example a String)